### DISCUSSION TITLE
thank you this helped me you should have more stars

### DISCUSSION THREAD

**Discussion by River-Mochi** [JakubAndrysek/mkdocs-open-in-new-tab#1] [General]
you should have more stars. I gave you one.
this helped me bc it was easy to just add this to YML file and it works. 
I did not know how to make the special commands to make something like this work correctly without your plugin. 

 this plugin should be included in materials mkdocs :)

thanks

**Comment by JakubAndrysek** [comment]
Thank you for your reply. I have added a basic description of how the plugin works.
Have I changed it correctly? Do you want to know more information from the readme?

**Reply by River-Mochi** [reply]
> Thank you for your reply. I have added a basic description of how the plugin works. Have I changed it correctly? Do you want to know more information from the readme?

I think I wrote it wrong.  your plugin was simple and easy to use even for a beginner. 
I was saying it was too complex on my own to to figure out how to code this kind of thing. only because I discovered  your plugin,  it was 2 steps and 2 min later it totally worked 👍 Excellent for normal  people like me who are focused more on content creation. 

 Materials- Mkdocs people  should consider including your plugin as part of their default package since it's so simple and useful  :)

**Comment by Wcowin** [comment]
Thank you very much for your work, I have been using it on [my blog](https://wcowin.work/)！

> Thank you for your reply. I have added a basic description of how the plugin works. Have I changed it correctly? Do you want to know more information from the readme?
